gee_tools/datasources/nonoptical_datasources.py

```diff
@@ -87,6 +88,7 @@ def build_img_coll(self):
         self.nl = ee.ImageCollection(self.nl)
     
     def get_img_coll(self):
+        # TODO: not sure why this is helpful. All it's doing is returning a class method (George).
```

Comment:
This method must be implemented for all ImageDatasource subclasses.  I think we can debate on what the right interface is.